src/components/Licenses/SBOMViewer.tsx

```
@@ -182,19 +177,12 @@ const BOMDisplay: React.FC = () => {
-              Copy BOM JSON
-            </button>
-          </CopyToClipboard>
-          {copyStatus && (
-            <span className="mt-2 block text-sm text-gray-600">
-              Copied to clipboard!
-            </span>
-          )}
+          <CopyButton
+            content={JSON.stringify(bomData, null, 2)}
+            tooltipContent="Copy BOM JSON to clipboard"
+            children="Copy BOM JSON"
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)